cpp_ext/TorchTensor.cpp

```diff
@@ -389,6 +389,18 @@ void PyAnyTorchTensorValue::bindDerived(ClassTy &c) {
                          &DefaultingPyInsertionPoint::resolve());
         });
 
+  // @overload double(self) -> Tensor
+  c.def("double",
+        [](const PyAnyTorchTensorValue &self) -> PyAnyTorchTensorValue {
+          const PyTorch_IntValue dtype = 7;
+          const PyTorch_BoolValue non_blocking = false;
+          const PyTorch_BoolValue copy = false;
+          const PyAnyTorchOptionalIntValue memory_format = NULL;
```

Comment:
it would be better to implement these kwargs with these defaults - like right now (today) we can probably be sure these are the right values but later on that might change.

Reply:
Got it. Including the dtype?

Reply:
> Got it. Including the dtype?

nah that one probably does make sense to hardcode. Also btw `NULL` isn't really a C++ thing, I believe it's "legal" because C++ is a "superset" of C, but I don't think you're supposed to use it. In general people use `nullptr` but I believe the correct thing here is `py::none()`.

Reply:
So according to how PyTorch is implemented, it seems like to() function's default values are identical to what I've hardcoded. However, I assume what you meant by default is the values of kwargs variables passed into this function, correct? 
https://pytorch.org/docs/stable/generated/torch.Tensor.to.html#torch.Tensor.to

Reply:
Changed memory_format to py::none().
Going through how PyTorch's torch.tensor.double() is implemented, and how the original print is like (i.e. I believe it's the default values), I didn't change defining non_blocking and copy as false. 

If this is not the correct approach, I can try modifying `PyAnyTorchTensorValue to(` in TorhcOps.h by adding default parameters to the function declaration, or follow your instruction. 

https://pytorch.org/docs/stable/generated/torch.Tensor.double.html
https://pytorch.org/docs/stable/generated/torch.Tensor.to.html#torch.Tensor.to
```
module attributes {torch.debug_module_name = "MySimpleTestModule"} {
  func.func @forward(%arg0: !torch.vtensor<[?,?,?],si64>) -> !torch.vtensor<[?,?,?],f64> {
...
    %5 = torch.aten.broadcast_to %0, %4 : !torch.vtensor<[],si64>, !torch.list<int> -> !torch.vtensor<[?,?,?],si64>
    %6 = torch.aten.to.dtype %5, %int7, %false, %false, %none : !torch.vtensor<[?,?,?],si64>, !torch.int, !torch.bool, !torch.bool, !torch.none -> !torch.vtensor<[?,?,?],f64>
    return %6 : !torch.vtensor<[?,?,?],f64>
  }
}
```

Reply:
implementation is fine but I believe you formatted the `cpp_ext/TorchTensor.pybinds.cpp` file which you shouldn't (otherwise we would have to format it each time). You can just regenerate it without the formatting.